### PR TITLE
Chore: Add ingestion limit per knowledge set, tweak knowledge UI

### PR DESCRIPTION
**Pull request by StrongMonkey**
- Add ingestion limit for knowledge set(1000)
- Only keep links in onedrive/website add modal
- Add scrape limit for website(and UI)

https://github.com/otto8-ai/tools/pull/202

### CODE REVIEW

File: pkg/controller/handlers/knowledgefile/knowledgefile.go [hunk 1]
```diff
@@ -30,12 +32,19 @@ func (u *UnsupportedError) Error() string {
 type Handler struct {
 	invoker   *invoke.Invoker
 	gptScript *gptscript.GPTScript
+	limit     int
 }
 
 func New(invoker *invoke.Invoker, gptScript *gptscript.GPTScript) *Handler {
+	defaultLimit := env.VarOrDefault("OTTO_KS_INGESTION_LIMIT", "1000")
```
Comment (thedadams) [inline pkg/controller/handlers/knowledgefile/knowledgefile.go]: Can this be an actual configuration parameter? In other words, can this be added here: https://github.com/otto8-ai/otto8/blob/0582592a8fc318a1ce96d60218e43df0e34a040f/pkg/services/config.go#L50-L61

Then passed as a parameter to this function.

File: ui/admin/app/components/knowledge/AddSourceModal.tsx [hunk 2]
```diff
@@ -161,146 +124,27 @@ const AddSourceModal: FC<AddSourceModalProps> = ({
                                         ? "Enter website URL"
                                         : "Enter OneDrive folder link"
                                 }
-                                className="w-[250px] dark:bg-secondary"
+                                className="w-[250px] dark:bg-secondar"
```
Comment (ryanhopperlowe) [inline ui/admin/app/components/knowledge/AddSourceModal.tsx]: looks like a typo here

**Review by ryanhopperlowe — COMMENTED** [review]
just fix the linter errors and you should be good

**Review by ryanhopperlowe — APPROVED** [review]

File: pkg/controller/handlers/knowledgefile/knowledgefile.go (hunk 1 above)
Comment (StrongMonkey) [reply pkg/controller/handlers/knowledgefile/knowledgefile.go]: yeah sure

File: ui/admin/app/components/knowledge/AddSourceModal.tsx (hunk 2 above)
Comment (StrongMonkey) [reply ui/admin/app/components/knowledge/AddSourceModal.tsx]: 👍 my bad

**Pull request merged**